README.md

```diff
@@ -109,41 +111,62 @@ app.run()                     # opens a window and starts the game.
 
    To close the window, you can by default, press shift+q or press the red x. to disable this, write `window.exit_button.enabled = False` somewhere in your code.
 
+## Documentation
+If you need help, you can watch documentation sites:
+   * [Ursina Documentation](https://www.ursinaengine.org/documentation.html)

```

Comment:
all links to the ursina webpage should be pokepetter.github.io/ursina instead of www.ursinaengine.org, in case the domain name expires.

Reply:
Fixed it